repositories.bzl

```diff
@@ -48,18 +48,6 @@ def build_bazel_rules_nodejs_dev_dependencies():
 
     build_bazel_rules_nodejs_dependencies()
```

Comment:
`build_bazel_rules_nodejs_dependencies` still needed?

Reply:
oh I see. since we're hoisting the nodejs workspace up here I suppose these should be renamed to
`rules_nodejs_dev_dependencies` and `rules_nodejs_dependencies`

Reply:
It will be replaced/merged with the `rules_nodejs` one. I have that pending locally atm.